package.json

```diff
@@ -17,7 +17,8 @@
         "storybook": "start-storybook -p 6006",
         "storybook:build": "build-storybook -c .storybook -o .storybook-static",
         "test:e2e": "cypress open",
-        "env": "wp-env"
+        "env": "wp-env",
+        "setup-e2e-tests-env": "npm install -D @wordpress/env@^2.1.0 puppeteer@^1.14.0 cypress@^6.1.0 jest@^24.5.0 jest-puppeteer@^4.1.0 --no-save"
```

Comment:
Why? Why not just have these as dev dependencies?

Reply:
@JasonTheAdams I added details in `Description` section, did you check?

Reply:
Ah, I did miss that. Honestly, I'd prefer to just keep the dependencies all in place. Doing it that way makes it less clear and the package-lock file is not able to make sure all dependencies are handled locked. Each time this script is run it would load another version (if available) of the packages. Please remove that script and include the packages as dev dependencies.

Reply:
@JasonTheAdams Why can't we use them when we need them. Why need to force to install these big dependencies on every `npm install`. I agree that their version will not store in a package lock file but will it break anything if we have a different minor version?

Reply:
@JasonTheAdams you are right about npm package caching. I struggled a lot yesterday when running `npm install`and because e2e setup requirement, npm was taken a lot of time to complete (may be a network issue). I come with this solution because of this experience.

Reply:
Keep in mind you were in a unique position yesterday, too, where you were fiddling around with the packages and re-installing. This is not a normal circumstance. I want to design our systems around the rule, not the exception.

Reply:
@JasonTheAdams I agree with you. I will add e2e environment related packages as dev dependencies.

Reply:
fixed: https://github.com/impress-org/givewp/compare/af04b550b31c...36f51737b97e